### PR TITLE
test(mxp): improving naming

### LINKED ISSUE #1605: test(mxp): minor fix to `triggerNonTrivialOrNoop`

### PR DESCRIPTION
**Pull request by lorenzogentile404**

### CODE REVIEW

**Comment by OlivierBBB** [comment]
I'm not understanding what the issue was that you corrected @lorenzogentile404

File: arithmetization/src/test/java/net/consensys/linea/zktracer/module/mxp/MxpTest.java [hunk 1]
```diff
@@ -312,11 +313,12 @@ private void triggerNonTrivialOrNoop(BytecodeCompiler program, boolean isHalting
     OpCode opCode;
 
     if (!isHalting) {
-      mxpType = MxpType.values()[util.nextRandomInt(1, 5)]; // Type 1 to 4 excluding opCodeLast
-      opCode = getRandomOpCodeByType(mxpType);
+      mxpType =
+          MxpType.values()[util.nextRandomInt(1, 5)]; // Type 1 to 4 excluding opCodesType4Halting
+      opCode = getRandomNonHaltingOpCodeByType(mxpType);
```
Comment (OlivierBBB) [inline arithmetization/src/test/java/net/consensys/linea/zktracer/module/mxp/MxpTest.java:320]: The function body of `getRandomNonHaltingOpCodeByType` didn't change. Was it about renaming it ?

File: arithmetization/src/test/java/net/consensys/linea/zktracer/module/mxp/MxpTest.java [hunk 2]
```diff
@@ -335,8 +337,11 @@ private void triggerNonTrivialOrNoop(BytecodeCompiler program, boolean isHalting
       offset1 = EWord.of(util.getRandomBigIntegerByBytesSize(0, MAX_BYTE_SIZE));
       offset2 = EWord.of(util.getRandomBigIntegerByBytesSize(0, MAX_BYTE_SIZE));
 
-      // NOOP case (except for Type2 and Type3 instructions)
-      if (mxpType != MxpType.TYPE_2 && mxpType != MxpType.TYPE_3) {
+      // NOOP case (except for Type2 and Type3 instructions and halting instructions,
+      // as we do not want halting instructions to turn into NOOPs)
```
Comment (OlivierBBB) [inline arithmetization/src/test/java/net/consensys/linea/zktracer/module/mxp/MxpTest.java]: I don't see why we wouldn't want NOOP's triggered by halting instructions. I would imagine we want to test all scenarios, this being one of them. Maybe in a separate test, but it's still of interest to have all ways of triggering a NOOP.

File: arithmetization/src/test/java/net/consensys/linea/zktracer/module/mxp/MxpTest.java (hunk 1 above)
Comment (lorenzogentile404) [reply arithmetization/src/test/java/net/consensys/linea/zktracer/module/mxp/MxpTest.java:320]: Yes, I just made the name more precise.

File: arithmetization/src/test/java/net/consensys/linea/zktracer/module/mxp/MxpTest.java (hunk 2 above)
Comment (lorenzogentile404) [reply arithmetization/src/test/java/net/consensys/linea/zktracer/module/mxp/MxpTest.java]: Ok.

**Review by OlivierBBB — APPROVED** [review]

**Pull request merged**